src/main/java/com/vizor/unreal/convert/Converter.java

```diff
@@ -63,10 +64,37 @@ public void convert(final Path srcPath, final List<Tuple<Path, Path>> paths)
         if (!Config.get().isNoFork())
             pathStream = pathStream.parallel();
 
-        pathStream.forEach(t -> {
-            log.info("Converting {}", t.first());
-            convert(srcPath, t.first(), t.second());
-        });
+
+
+			try {
+				List<ProtoFileElement> protos = new ArrayList<>();
+				List<ProtoProcessorArgs> args = new ArrayList<>();
+for (Tuple<Path, Path> pathPair : paths) {
+	final Path pathToProto = pathPair.first();
+	final Path pathToConverted = pathPair.second();
+	final String fileContent = join(lineSeparator(), readAllLines(pathPair.first()));
+	args.addAll(preProcess(parse(get(pathPair.first().toString()), fileContent)).stream().map(protoFile -> {
+		
+		final Path relativePath = srcPath.relativize(pathToProto); 
+		return new ProtoProcessorArgs(protoFile, relativePath, pathToConverted, moduleName);
+	}).collect(Collectors.toList()));
+}
+args.forEach(arg -> {
+	
+log.info("Converting {}", arg.pathToProto);
+	
+	new ProtoProcessor(arg, args).run();});
+				
+		// pathStream.flatMap(t ->
+		// ).collect(Collectors.toList());
+
+        // pathStream.forEach(t -> {
+        //     log.info("Converting {}", t.first());
+        //     convert(srcPath, t.first(), t.second());
+		// });
+	}
+	catch (IOException ex)
+	{}
```

Comment:
Unhandled exception. you can just throw new RuntimeException or at least log error message to have to have an idea what happens if something went wrong

Reply:
This is from a past version, I've since (in 52b73a5) changed this exception silencing in favor of throwing a RuntimeException.